<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TSettingDlg</class>
 <widget class="QDialog" name="TSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>母材设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QCheckBox" name="mOOBBRotCBox">
       <property name="text">
        <string>主轴旋转</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>母材高度</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>排版网格间距</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="mRot90CBox">
       <property name="text">
        <string>90</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QCheckBox" name="mRot180CBox">
       <property name="text">
        <string>180</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="mRot270CBox">
       <property name="text">
        <string>270</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QDoubleSpinBox" name="mWidthSBox">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
       <property name="value">
        <double>3000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QDoubleSpinBox" name="mDeltaSBox">
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>母材宽度</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QDoubleSpinBox" name="mHeightSBox">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
       <property name="value">
        <double>2000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>迭代优化次数</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QSpinBox" name="mMaxIterSBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOkBtn">
       <property name="text">
        <string>确定(&amp;O)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelBtn">
       <property name="text">
        <string>取消(&amp;C)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mWidthSBox</tabstop>
  <tabstop>mHeightSBox</tabstop>
  <tabstop>mDeltaSBox</tabstop>
  <tabstop>mOkBtn</tabstop>
  <tabstop>mCancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
